ci/copy-unix-release-artifacts.sh

```diff
@@ -9,15 +9,19 @@ OUTPUT_DIR=$3
 
 mkdir -p $OUTPUT_DIR/github
 mkdir -p $OUTPUT_DIR/artifactory
+# Artifacts that we only use in the split-release process
+mkdir -p $OUTPUT_DIR/split-release
 
 
 TARBALL=daml-sdk-$RELEASE_TAG-$NAME.tar.gz
 EE_TARBALL=daml-sdk-$RELEASE_TAG-$NAME-ee.tar.gz
+bazel build //release:sdk-release-tarball-ce //release:sdk-release-tarball-ee
```

Comment:
Why are those suddenly needed? Local testing?

Reply:
yeah just local testing. seemed easier to build everything than to rely on stuff having been built before by a CI job.